src/libs/actions/Workflow.ts

```diff
@@ -327,7 +327,7 @@ function clearApprovalWorkflowApprovers() {
     Onyx.merge(ONYXKEYS.APPROVAL_WORKFLOW, {approvers: []});
 }
 
-function setApprovalWorkflow(approvalWorkflow: ApprovalWorkflowOnyx) {
+function setApprovalWorkflow(approvalWorkflow: OnyxMergeInput<typeof ONYXKEYS.APPROVAL_WORKFLOW>) {
```

Comment:
Why was this type added? I don't think it's used anywhere else in App, is it?

Reply:
It was my [call](https://github.com/Expensify/App/pull/47642#discussion_r1722879996), I think it is more accurate than changing errors type as before 😄